### PR TITLE
TASK: Adjust to rewrite of `_hiddenInIndex` to `hiddenInMenu`

### PR DESCRIPTION
**Pull request by kitsunet**
Related: https://github.com/neos/neos-development-collection/pull/4921

### CODE REVIEW

File: NodeTypes/Content/Registration/Registration.fusion [hunk 1]
```diff
@@ -6,7 +6,7 @@ prototype(Neos.Demo:Content.Registration) < prototype(Neos.Neos:ContentComponent
     renderer = Neos.Fusion.Form:Runtime.RuntimeForm {
         namespace = "neos-demo-registration"
         attributes {
-            id = ${'form-' + node.identifier}
+            id = ${'form-' + node.property('_identifier')}
```
Comment (mhsdesign) [inline NodeTypes/Content/Registration/Registration.fusion]: as far as i know this behaviour is deprecated and `node.nodeAggregateId.value` should be used? Or we could have another flowquery operation for this? `q(node).identifier()`

File: Resources/Private/Fusion/Integration/Footer.fusion [hunk 2]
```diff
@@ -13,6 +13,7 @@ prototype(Neos.Demo:Integration.Footer.Renderer) < prototype(Neos.Demo:Presentat
     menuItems = Neos.Fusion:Map {
         items = ${q(site).property('metaNavigationItems')}
         itemRenderer = Neos.Fusion:DataStructure {
+            depth = ${node.property('_depth')}
```
Comment (mhsdesign) [inline Resources/Private/Fusion/Integration/Footer.fusion]: `_depth` is also deprecated and should better not be used in the field.
Still deprecated but better is `\Neos\Neos\Fusion\Helper\NodeHelper::depth`

File: NodeTypes/Content/Registration/Registration.fusion (hunk 1 above)
Comment (bwaidelich) [reply NodeTypes/Content/Registration/Registration.fusion]: we decided to go for `q(node).id()` IIRC

File: Resources/Private/Fusion/Integration/Footer.fusion (hunk 2 above)
Comment (bwaidelich) [reply Resources/Private/Fusion/Integration/Footer.fusion]: why is the depth needed here at all?

**Comment by kitsunet** [comment]
yep this is all old and needs to be redone now :) I take care

File: Resources/Private/Fusion/Integration/Footer.fusion (hunk 2 above)
Comment (kitsunet) [reply Resources/Private/Fusion/Integration/Footer.fusion]: It isn't, I think it was in the past? IDK, removing

**Review by mhsdesign — APPROVED** [review]

**Review by ahaeslich — APPROVED** [review]

**Pull request merged**